src/graphics/aurora/cursor.cpp

```diff
@@ -112,6 +114,12 @@ void Cursor::load() {
 
 	TXI *txi = new TXI();
 	txi->getFeatures().filter = false;
+	
+	//if we need DeS3TC, we decompress the cursor image
+	if(GfxMan.needManualDeS3TC())
```

Comment:
Please put a space after the if, and add the opening brace to the same line

`if (GfxMan.needManualDeS3TC()) {`